### ISSUE TITLE
Use the log crate instead of print!

### ISSUE THREAD

**Issue by rnestler** [coredump-ch/coredumpbot#18]
https://crates.io/crates/log

**Comment by dns2utf8** [comment]
Can we assign this to #19 ?

**Comment by rnestler** [comment]
What do you mean with _assign_?

**Comment by dns2utf8** [comment]
Combine the Issue with the PR

**Comment by rnestler** [comment]
I don't know how to do it but @dbrgn told me it's possible.

**Comment by dbrgn** [comment]
You can't do that after the PR is already open.

But if you have [hub](https://github.com/github/hub) installed, you could have converted this issue to a pull request:

```
hub pull-request -b master -i 18 -h use_log_crate
```

I created a shortcut function in my bashrc:

```
function convert-pull-request() {
    if [ $# -eq 3 ]; then
        echo "hub pull-request -b $1 -i $2 -h $3"
    elif [ $# -eq 2 ]; then
        echo "hub pull-request -b $1 -i $2"
    else
        echo "Usage: convert-pull-request <base> <issue-nr> [<branch>]"
    fi
}
```

You can pipe the output into bash to execute.

**Comment by rnestler** [comment]
Fixed with #19
